src/frontend/hooks/persistedState/createPersistedState.ts

```diff
@@ -15,7 +15,8 @@ type PersistedStoreKey =
   | 'MapeoTrack'
   | 'Passcode'
   | 'ActiveProjectId'
-  | 'Settings';
+  | 'Settings'
+  | 'Permissions';
```

Comment:
Nit: I could see giving this another name because we plan to add additional permissions in the future. For example, for other kinds of opt-in metrics.